historyAPI/router.js

```diff
@@ -0,0 +1,74 @@
+export function CreateRouter() {
```

Comment:
한수님 지금코드는 function 보다는 class로 만들어주는게 좋을거 같아요.
```suggestion
export class Router{
```

Reply:
https://github.com/choihansu97/flab-review/pull/7/commits/4c6d9f70cdb8e97e98a9cd1d6f83f0d57673ad8e
수정했습니다.